streaming/streaming.go

```diff
@@ -287,3 +287,24 @@ func LaunchDataflowJob(ctx context.Context, targetProfile profiles.TargetProfile
 		" will have to be manually cleaned up via he UI. HarbourBridge will not delete them post completion of the migration.")
 	return nil
 }
+
+func StartDatastream(ctx context.Context, sourceProfile profiles.SourceProfile, targetProfile profiles.TargetProfile) (StreamingCfg, error) {
+	streamingCfg, err := ReadStreamingConfig(sourceProfile.Conn.Mysql.StreamingConfig, targetProfile.Conn.Sp.Dbname)
```

Comment:
Consider adding a funciton to multiplex the streaming config, shouldn't be too much effort, but can potentially finish both flows for oracle and mysql? We can return an error for the other type of sources

Reply:
We don't need to return an error for other types of sources as this function is called only for oracle and mysql. Implementation for other databases is empty. 
I will multiplex the streaming config here based on the source profile connection. If connection is mysql then StreamingConfig for the same will be used and vice versa for oracle.